python/tvm/relay/prelude.py

```diff
@@ -0,0 +1,114 @@
+# pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name
+"""Include some preloaded term/type definitions."""
+from .ty import GlobalTypeVar, TypeVar, FuncType
+from .expr import Var, Function, GlobalVar
+from .adt import Constructor, TypeData, Clause, Match
+from .adt import PatternConstructor, PatternVar, PatternWildcard
+
+class Prelude:
```

Comment:
@MarisaKirisame can you add docs for the standard ADTs? It is not clear, e.g., why Peano nats are necessary (if it's just for our tests, we should leave it in testing files)

Reply:
I guess it might be worth then deciding what _should_ be the standard ADTs in the prelude and what should be kept in the test file. Lists, I think, are a very reasonable one to have

Reply:
Agree.